src/style.css

```
@@ -0,0 +1,177 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 198px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  height: 134px;
+  width: 160px;
+
+  margin-top: 32px;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+}
+
+.product__name {
+  margin-top: 40px;
+  margin-bottom: 0;
+
+  width: 166px;
+  height: 36px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.product__code {
+  margin-top: 4px;
+  margin-bottom: 0;
+
+  font-size: 10px;
+  font-weight: 400;
+
+  color: #616070;
+}
+
+.review {
+  margin-top: 16px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
```

`direction: row` is default;
to use the benefits of flexbox:
```suggestion
  width: 100%;
  justify-content: space-between;
```